docs/user-guide/server-side-install-overview.md

```diff
@@ -1,19 +1,122 @@
+
+Select one of the following methods to install the Zowe server-side components on z/OS. 
+<!-- Add the advantages and disadvantanges to each method -->
+
+   - **Convenience build**
+
+     Zowe z/OS binaries are packaged as a PAX file which is a full product install. Transfer this PAX file to a USS directory and expand its contents. Use the [zwe](installandconfig.md#zwe-server-command) command `zwe install` to extract a number of PDS members which contain load modules, JCL scripts, and PARMLIB entries. 
+
+     To obtain and install the convenience build, see [Installing Zowe runtime from a convenience build](install-zowe-zos-convenience-build.md).
+
+   - **SMP/E build**
+
+     The Zowe z/OS binaries are packaged as the following files that you can download. You install this build through SMP/E.  
+     - A pax.Z file, which contains an archive (compressed copy) of the FMIDs to be installed.
+     - A readme file, which contains a sample job to decompress the pax.Z file, transform it into a format that SMP/E can process, and invoke SMP/E to extract and expand the compressed SMP/E input data sets.
+
+     To obtain and install the SMP/E build, see [Installing Zowe SMP/E](install-zowe-smpe.md).
+
+
+   - **Portable Software Instance (PSWI)**
+
+     You can acquire and install the Zowe z/OS PAX file as a portable software instance (PSWI) using z/OSMF.
+
+     To obtain and install the PSWI, see [Installing Zowe from a Portable Software Instance](install-zowe-pswi.md).
+
+   While the procedures to obtain and install the convenience build, SMP/E build, or PSWI are different, configuration of a Zowe runtime is the same for all three installation methods.
+
+Successful installation of either a convenience build or an SMP/E build creates the following artifacts:
+* A zFS folder that contains the unconfigured Zowe runtime directory
+* A utility library `SZWEEXEC` that contains utilities
+* A SAMPLIB library `SZWESAMP` that contains sample members
+* A load library `SZWEAUTH` that contains load modules. 
+
+The steps to prepare the z/OS environment to launch Zowe are the same for all installation methods.
+
+## Roles and responosibilities in server-side component installation process
+
+### Tasks performed by the security administrator
+
+To configure Zowe security for production environments, it is likely that your organization's security administrator will be required to perform various tasks. For more information,see [Address security requirements](./address-security-requirements#tasks-performed-by-your-security-administrator).
+
+### Tasks performed by the storage administrator
+
+Before starting installation, notify the storage administrator to reserve enough space for USS, directory storage space, and any other storage requrements to install Zowe.
+
+### Tasks performed by the network administrator
+
+Notify your organization's network administrator to assign port numbers, reserve these port numbers, and arrange them for you.
```

Comment:
Maybe mentioning HA sysplex distributor for high availability scenarios is going to much into detail?